### PR TITLE
feat: add searchbox docs

### PR DESCRIPTION
**Pull request by jyash97**

### CODE REVIEW

File: content/docs/reactivesearch/searchbox/Quickstart.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: 'QuickStart'
+meta_title: 'QuickStart to SearchBox'
+meta_description: 'SearchBox is a lightweight search library with some common utilities.'
+keywords:
+    - quickstart
+    - react-searchbox
```
Comment (ShahAnuj2610) [inline content/docs/reactivesearch/searchbox/Quickstart.md]: This should be searchbox.
And also I think you forgot to add `Link` in `NavBar.js`

File: content/docs/reactivesearch/searchbox/Quickstart.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'searchbox'
+---
+
+[searchbox](https://github.com/appbaseio/searchbox) - A lightweight search library with some common utilities.
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/searchbox/Quickstart.md]: @jyash97 See the description for React Searchbox. This is for ElasticSearch, that's not clear in your description.

File: content/docs/reactivesearch/searchbox/Quickstart.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+## Installation
+
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/searchbox/Quickstart.md:18]: @jyash97 Explain why a user needs to install both.

File: content/docs/reactivesearch/searchbox/api.md [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+    - searchbox
+    - search library
+    - elasticsearch
+sidebar: 'docs'
+nestedSidebar: 'searchbox'
+---
+
+![Image to be displayed](https://i.imgur.com/rxOM8Ut.png)
+
+`SearchBox` offers a lightweight and performance focused searchbox UI component to query and display results from your ElasticSearch app (aka index).
```
Comment (siddharthlatest) [inline content/docs/reactivesearch/searchbox/api.md:16]: @jyash97 See, this description is fine. The other page has a different one.

**Review by siddharthlatest — CHANGES_REQUESTED** [review]

File: content/docs/reactivesearch/searchbox/Quickstart.md (hunk 1 above)
Comment (jyash97) [reply content/docs/reactivesearch/searchbox/Quickstart.md]: Thanks good catch 🥇

**Review by siddharthlatest — APPROVED** [review]

**Pull request merged**